test/BasicEditingTest.VI.cs

```diff
@@ -1121,5 +1121,18 @@ public void ViInsertModeMoveCursor()
                 _.RightArrow, // 'RightArrow' again does nothing, but doesn't crash
                 "c"));
         }
+
+        [SkippableFact]
+        public void ViDefect1281()
+        {
+            TestSetup(KeyMode.Vi);
+
+            Test("bcd", Keys(
+                "abcdabcd", _.Escape,
+                "Bc2tb", _.Escape, CheckThat(() => AssertCursorLeftIs(0)), 
+                "c2tb", CheckThat(() => AssertLineIs("bcd")),
+                "c2tb" // should not crash
```

Comment:
Can you please add comment about what each input sequence does in VI mode? For example, a comment about what 
functions will be triggered by `Bc2tb`.

Reply:
✔

Reply:
Same here too :) command not added in the latest changes.

Reply:
Could not find the code – so that must have been my mistake.
Sorry for the confusion. 🤦‍♂️
This is pushed now.